css/style.css

```diff
@@ -0,0 +1,615 @@
+*,
+*::after,
+*::before {
+  box-sizing:border-box }
+
+body {
+  max-width: 1350px;
+  margin: 0 auto!important;
+  background-color: rgb(253, 249, 249)!important;
+}
+
+html, body {
+    overflow-x: hidden;
+}
+.container-fluid {
+    position: relative;
+    background-color: white
+}
+
+.heading-none{
+    display: none
+}
+
+.nav-bar {
+	position: relative;
+	margin: auto;
+	background-color: #f72210;
+	color: #fff;
+	padding: 50px 0px 10px 0px;
+	max-width: 1310px;
+}
+
+
+.nav-second {
+	position: relative;
+	max-width: 1310px;
+	margin: auto;
+}
+
+.logoimg {
+	position: absolute;
+	width: 396px;
+	height: 60px;
+	transform: translate(-50%, 0);
+	top: 15px;
+	left: 50%;
+}
+.logoimg > img {
+	width: 100%;
+}
+
+.datenav {
+	position: relative;
+	margin-left: 20px;
+	font-size: 16px;
+}
+
+.signin {
+	margin-right: 20px;
+	font-size: 18px;
+	font-weight: 600;
+	position: relative;
+	bottom: 10px;
+}
+
+.signin a {
+	text-decoration: none;
+	color: #f72210;
+	background-color: #fff;
+	padding: 8px;
+}
+
+.fa-chevron-right {
+	padding-right: 10px;
+	padding-left: 10px;
+	z-index: 9999;
+}
+
+.signin span {
+	padding-right: 10px;
+}
+
+.header-bottom {
+    margin-top: 100px;
+	color: #000;
+	height: 50px;
+	padding: 10px 0px;
+	background-color: #fff;
+	border-bottom: 1px solid #d9d9d9;
+	position: relative;
+}
+
+.header-bottom::before {
+	width: 66px;
+	position: absolute;
+	top: 0;
+	right: 0;
+	content: "";
+	display: block;
```

Comment:
`SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.